hr_holidays_public/models/hr_leave.py

```diff
@@ -0,0 +1,21 @@
+# Copyright 2017-2018 Tecnativa - Pedro M. Baeza
+# Copyright 2018 Brainbean Apps
+# Copyright 2020 InitOS Gmbh
```

Comment:
You mustn't add any copyright to a file that you don't touch (undo it for all the untouched files).